src/app/pages/evento/evento.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { EventoService } from '../../services/evento.service';
+import { ActivatedRoute } from '@angular/router';
+import { Evento } from 'src/app/models/evento';
+import { UsuarioService } from 'src/app/services/usuario.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138940)
```suggestion
import { UsuarioService } from "src/app/services/usuario.service";
```